market_scanner.py

```diff
@@ -8,26 +8,30 @@
 import sys
 from stocklist import NasdaqController
 from tqdm import tqdm
-
 from joblib import Parallel, delayed, parallel_backend
 import multiprocessing
 
-def show_exception_and_exit(exc_type, exc_value, tb):
-    import traceback
-    traceback.print_exception(exc_type, exc_value, tb)
-    input("Press key to exit.")
-    sys.exit(-1)
 
+###########################
+# THIS IS THE MAIN SCRIPT #
+###########################
+
+# Change variables to your liking then run the script
+MONTH_CUTTOFF = 6
```

Comment:
Instead of running around with Globals, we probably want to have a class called config or something. On startup, it's populated with the default values seen here, unless there are arguments

https://www.tutorialspoint.com/python/python_command_line_arguments.htm

This allows you to have one script running with the defaults and the other running with some other behavior entirely, without needing two copies of the repo.